__tests__/foursquareSearch_test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import FoursquareSearch from '../src/javascripts/components/map/foursquareSearch/foursquareSearch';
+
+it('should search venues based on the value in the state', () => {
```

Comment:
So test checks whether search works when the form is submitted

Then it's a good idea to check if `props.searchVenues` is called with `bar` term